radicale_imap/__init__.py

```diff
@@ -54,17 +82,11 @@ def is_authenticated(self, user, password):
         if sys.version_info < (3, 4) and secure:
             raise RuntimeError("Secure IMAP is not availabe in Python < 3.4")
         try:
-            connection = imaplib.IMAP4(host=address, port=port)
```

Comment:
Just do something like this, instead of creating two methods:

```python
if imaps and secure:
    connection = imaplib.IMAP4_SSL(...
elif imaps:
    connection = imaplib.IMAP4_SSL(...
else:
    connection = imaplib.IMAP4(host=address, port=port)
    try:
    ...